app/src/main/java/chat/rocket/android/chatrooms/ui/ChatRoomsFragment.kt

```diff
@@ -272,11 +272,13 @@ class ChatRoomsFragment : Fragment(), ChatRoomsView {
     }
 
     private fun setupListeners() {
-        text_server_name.setOnClickListener {
-            ServersBottomSheetFragment().show(
-                activity?.supportFragmentManager,
-                chat.rocket.android.servers.ui.TAG
-            )
+        if (getString(R.string.server_url).isEmpty()) {
+            text_server_name.setOnClickListener {
+                ServersBottomSheetFragment().show(
+                    activity?.supportFragmentManager,
```

Comment:
Thank you very much for sending this PR @cardoso! I'm glad to see you here.

Just a tip: `chat.rocket.android.servers.ui.TAG` fits on the same line where `activity?.supportFragmentManager` is declared here - respecting the coding style. But it's fine!

Thank you! 👊